solutions/security/ai/triage-alerts.md

```diff
@@ -60,3 +59,43 @@ After you review the report, click **Add to existing case** at the top of AI Ass
 :alt: An AI Assistant dialogue with the add to existing case button highlighted
 :screenshot:
 :::
+
+
+## Example alert triage workflow
+
+This section shows an example workflow for triaging a specific alert.
+
+**Scenario:** You are investigating an alert: "Multiple Failed Logins Followed by Success - user: jsmith"
+
```

Comment:
This might be a good place to use the stepper component: https://elastic.github.io/docs-builder/syntax/stepper/